manifests/d/dev47apps/DroidCam/6.4.3/dev47apps.DroidCam.installer.yaml

```diff
@@ -1,11 +1,21 @@
-# Created using wingetcreate
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
 PackageIdentifier: dev47apps.DroidCam
 PackageVersion: 6.4.3
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress

```

Comment:
```suggestion
```
For info:
Nullsoft is `interactive` and `silent` or just `silent` depending on the installer.
Inno usually uses all 3